tools/sofa_test_scene/diamondRobot.py

```diff
@@ -31,51 +29,90 @@
         }
     ]
 
+plugins=["SofaPython3","SoftRobots","ModelOrderReduction","STLIB",
+         # normally plugin Sofa.Component is enough but still warning
+         "Sofa.Component.Visual",
+         "Sofa.Component.AnimationLoop",
+         "Sofa.GL.Component.Rendering3D",
+         "Sofa.Component.Constraint.Lagrangian.Solver",
+         'Sofa.Component.IO.Mesh',
+         'Sofa.Component.Playback',
+         'Sofa.Component.Constraint.Lagrangian.Correction', # Needed to use components [GenericConstraintCorrection]
+         'Sofa.Component.Engine.Select', # Needed to use components [BoxROI]
+         'Sofa.Component.LinearSolver.Direct', # Needed to use components [SparseLDLSolver]
+         'Sofa.Component.Mapping.Linear', # Needed to use components [BarycentricMapping]
+         'Sofa.Component.Mass', # Needed to use components [UniformMass]
+         'Sofa.Component.ODESolver.Backward', # Needed to use components [EulerImplicitSolver]
+         'Sofa.Component.SolidMechanics.FEM.Elastic', # Needed to use components [TetrahedronFEMForceField]
+         'Sofa.Component.SolidMechanics.Spring', # Needed to use components [RestShapeSpringsForceField]
+         'Sofa.Component.StateContainer', # Needed to use components [MechanicalObject]
+         'Sofa.Component.Topology.Container.Dynamic'] # Needed to use components [TetrahedronSetTopologyContainer]
+
 meshPath = os.path.dirname(os.path.abspath(__file__))+'/mesh/'
 
 def createScene(rootNode):
 
+    rootNode.addObject('FreeMotionAnimationLoop')
+    rootNode.addObject('GenericConstraintSolver', tolerance="1e-6", maxIterations="1000")
+    rootNode.addObject('OglSceneFrame', style="Arrows", alignment="TopRight")
     rootNode.addObject('VisualStyle', displayFlags='showVisualModels showForceFields')
+    rootNode.addObject('RequiredPlugin', pluginName=plugins, printLog=False)
 
     rootNode.findData('gravity').value=[0.0,0.0,-9810];
     rootNode.findData('dt').value=1
 
-    plugins=["SofaPython","SoftRobots","ModelOrderReduction"]
-    for name in plugins:
-        rootNode.addObject('RequiredPlugin', name=name, printLog=False)
-        
-    rootNode.addObject('OglSceneFrame', style="Arrows", alignment="TopRight")
 
-    rootNode.addObject('FreeMotionAnimationLoop')
-    rootNode.addObject('GenericConstraintSolver', tolerance="1e-6", maxIterations="1000")
+    volumeMeshFileName=meshPath+'siliconeV0.vtu'
+    rotation=[90, 0.0, 0.0]
+    translation=[0.0, 0.0, 35]
+    totalMass=0.5
+    surfaceMeshFileName=meshPath+'surface.stl'
+    surfaceColor=[0.7, 0.7, 0.7, 0.7]
+    poissonRatio=0.45
+    youngModulus=450
+    scale=[1., 1., 1.]
+
+
+    modelNode = rootNode.addChild('modelNode')
+    modelNode.addObject('EulerImplicitSolver', name='integration')
+    modelNode.addObject('SparseLDLSolver', name="solver", template='CompressedRowSparseMatrixd')
```

Comment:
```suggestion
    modelNode.addObject('SparseLDLSolver', name="solver", template='CompressedRowSparseMatrixMat3x3d')
```
It should be faster if there are no rigids